lib/card_generator.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/card'
+
+class Card_Generator
```

Comment:
This class should be called `CardGenerator`. While the name of this **file** is conventional, Class names use Pascal Case in Ruby